runtime/runtime/src/lib.rs

```diff
@@ -281,138 +334,161 @@ impl Runtime {
         debug!(target: "runtime", "{}", log_str);
     }
 
-    fn parallel_validate_transactions(
-        config: &RuntimeConfig,
+    /// Processes a group of transactions (same signer_id and public_key), sorted by ascending nonce.
+    /// Validates, calculates cost, and performs checks sequentially.
+    /// If a transaction fails validation or verification, it invokes handle_invalid_transaction
+    /// and, if `RelaxedChunkValidation` is enabled, skips to the next transaction without failing
```

Comment:
where is relaxedchunkvalidation being used here?

Reply:
In `Self::handle_invalid_transaction` calls